package/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "include": ["./src/**/*"],
+    "compilerOptions": {
+        "rootDir": "./src",
+        "module": "amd",
+        "moduleResolution": "node",
+        "outDir": "./out/amd",
+        "target": "ES2020",
```

Comment:
Was es5. I'm assuming updating this to something a little newer is fine. It should make debugging a fair bit easier.

Reply:
r.js Failed while parsing because of object spread (`...`). I've updated it to the latest version it successfully parses.

Reply:
Need to keep the es5, it might break consumers (Synapse for sure, Ibiza portal, and more)

Reply:
Bummer. I've reverted the target to es5